Sources/MongoSwift/Operations/ListCollectionsOperation.swift

```diff
@@ -81,10 +81,19 @@ public struct ListCollectionsOptions: Codable {
     /// The batchSize for the returned cursor.
     public var batchSize: Int?
 
+    /// An arbitrary BSON type to help trace the operation through
+    /// the database profiler, currentOp and logs. The default is to not send a value.
+    public var comment: BSON?
```

Comment:
ditto adding a test for this

Reply:
Added test testCreateListDropIndexesComment() in MongoCollection+IndexTests